components/TextFields/src/MDCTextInputControllerLegacyDefault.m

```diff
@@ -219,6 +229,10 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
     _textInput = [aDecoder decodeObjectForKey:MDCTextInputControllerLegacyDefaultTextInputKey];
     _trailingUnderlineLabelTextColor = [aDecoder
         decodeObjectForKey:MDCTextInputControllerLegacyDefaultTrailingUnderlineLabelTextColor];
+    _underlineHeightActive = (float)[aDecoder
+        decodeFloatForKey:MDCTextInputControllerLegacyDefaultUnderlineHeightActiveKey];
+    _underlineHeightActive = (float)[aDecoder
+        decodeFloatForKey:MDCTextInputControllerLegacyDefaultUnderlineHeightNormalKey];
```

Comment:
Should there be a check for the value's existence before decoding it? I believe  a Storyboard/Xib will have no value and you'll end up with 0 instead of 1.

Reply:
Done.